test/mapmatch.cc

```diff
@@ -117,7 +117,7 @@ std::string json_escape(const std::string& unescaped) {
 int seed = 973;
 int bound = 81;
 std::string make_test_case(PointLL& start, PointLL& end) {
-  static std::default_random_engine generator(seed);
+  static std::minstd_rand0 generator(seed);
```

Comment:
`default_random_engine` has implementation specific charactersistics which means it can differ from one compiler (stdlib implementation) to the next.
cc @dnesbitt61

Reply:
Should other uses of default_random_engine be replaced?

Reply:
yes if we want determinism accross compilers/implementations of stdlib